CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.2.4 / 2021/06/01
+===================
+- use new `apify-shared` packages to reduce bundle size
```

Comment:
Have you checked how large the reduction was?

Reply:
hmmm currently it actually did not make any difference, because the original `apify-shared` is still transitive dependency of some packages like `browser-pool` or the SDK. if i remove it, we save ~1mb (from ~207mb), but the main benefit will be in removed dependencies, so i am expecting much more (lets say tens of mbs).

![](https://pbs.twimg.com/media/EYEgxfLXsAAItE9?format=jpg&name=small)

Reply:
I was talking about the `apify-client` webpack bundle size 😄